@xen-orchestra/immutable-backups/liftProtection.mjs

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+import fs from 'node:fs/promises'
+import * as Directory from './directory.mjs'
+import { dirname, join } from 'node:path'
+import { createLogger } from '@xen-orchestra/log'
+import { listOlderTargets  } from './fileIndex.mjs'
+import { load as loadConfig } from 'app-conf'
+
+const { warn } = createLogger('xen-orchestra:immutable-backups:liftProtection')
+
+const APP_NAME = 'xo-immutable-backups'
+const APP_DIR = new URL('.', import.meta.url).pathname
+
+async function liftImmutability(immutabilityCachePath, immutabilityDuration) {
+    for await (const { index, target } of listOlderTargets(immutabilityCachePath, immutabilityDuration)) {
+      await Directory.liftImmutability(target, immutabilityCachePath)
+      await fs.unlink(index)
+      if (target.match(/xo-vm-backups\/[^/]+\/[^/]+\.json$/)) {
+        // snipe vm metadata cache to force XO to update it
+        await fs.unlink(join(dirname(target), 'cache.json.gz'))
+      }
+    }
+  }
+
+async function main(){
+    const { remotes } = await loadConfig(APP_NAME, {
+        appDir: APP_DIR,
+        ignoreUnknownFormats: true,
+    })
+    
+    for (const [remoteId, {indexPath,immutabilityDuration}] of Object.entries(remotes)) {
+        const basePath = indexPath ?? process.env.XDG_DATA_HOME ?? join('~', '.local', 'share')
```

Comment:
Extract in `_indexPath.mjs`.

Reply:
I put the logic in the _loadConfig.mjs , my intention is that the config is either rejected or ready to use